### PR TITLE
fix(multiSelect): select multiple options on multiSelect component

### PR DESCRIPTION
**Pull request by Teren-Velan**
Improve on multiSelect component:
- added props from React-select libray to enable multiSelect bar does not close after first option clicked
- add alert feedback from chakraUI to inform user that they can choose more than one option

### CODE REVIEW

File: client/components/recycleAndReuseComponents/AddItemMultiSelect.js [hunk 1]
```diff
@@ -30,13 +63,21 @@ const AddItemMultiSelect = ({ data, setItems, openDialog }) => {
 		event.preventDefault();
 		const selectedItems = getSelectedItems(event.target.elements);
 		setItems(selectedItems);
-		openDialog();
+		willTriggerDialog(true);
 	};
 
 	return (
 		<form onSubmit={handleSubmit} style={{ width: "100%" }}>
+			{!!selectedOptions && selectedOptions.length ? (
```
Comment (ryhuz) [inline client/components/recycleAndReuseComponents/AddItemMultiSelect.js:71]: this block is the main added logic right? this block can stay

File: client/components/recycleAndReuseComponents/AddItemMultiSelect.js [hunk 2]
```diff
@@ -30,13 +63,21 @@ const AddItemMultiSelect = ({ data, setItems, openDialog }) => {
...
+				<Alert size="1rem" mb="1rem" status="success">
+					<AlertIcon />
+					Please feel free to select more than one item from the list.
+				</Alert>
+			) : null}
+
 			<Select
 				isMulti
+				closeMenuOnSelect={false}
```
Comment (ryhuz) [inline client/components/recycleAndReuseComponents/AddItemMultiSelect.js:80]: this block is part of the main added logic right? this block can stay

File: client/components/recycleAndReuseComponents/AddItemMultiSelect.js [hunk 3]
```diff
@@ -1,12 +1,42 @@
-import { Button, Flex, Text } from "@chakra-ui/react";
-import React from "react";
+import { Button, Flex, Text, Alert, AlertIcon } from "@chakra-ui/react";
```
Comment (ryhuz) [inline client/components/recycleAndReuseComponents/AddItemMultiSelect.js:1]: revert all other changes - only main logic change should remain (as mentioned below)

File: client/components/recycleAndReuseComponents/AddItem.js [hunk 4]
```diff
@@ -18,6 +18,7 @@ const AddItem = ({ setNextStep, data, setItems }) => {
 				data={data}
 				setItems={setItems}
 				openDialog={() => willTriggerDialog(true)}
+				willTriggerDialog={willTriggerDialog}
```
Comment (ryhuz) [inline client/components/recycleAndReuseComponents/AddItem.js:21]: revert all other changes - only main logic change should remain (as mentioned below)

**Review by ryhuz — CHANGES_REQUESTED** [review]

File: client/components/recycleAndReuseComponents/AddItemMultiSelect.js (hunk 1 above)
Comment (Teren-Velan) [reply client/components/recycleAndReuseComponents/AddItemMultiSelect.js:71]: hey sorry for the delay , yup ok not sure why theres alot of other changes also , will close this PR and redo this portion

**Pull request closed without merging**